build.gradle.kts

```diff
@@ -64,28 +65,32 @@ tasks {
                 )
             }
         }
-    }
 
-    runIde {
-        maxHeapSize = "8g"
-    }
-
-    buildSearchableOptions {
-        enabled = false
+        ideaVersion {
+            sinceBuild = providers.gradleProperty("pluginSinceBuild")
+            untilBuild = providers.gradleProperty("pluginUntilBuild")
+        }
     }
 
-    sourceSets {
```

Comment:
This needs to be brought back.
At the same time, you can replace it with:
```
sourceSets {
    test {
        resources {
            setSrcDirs(listOf("src/test/fixtures"))
        }
    }
}
```

Reply:
fixed